src/Primitive/Listt.php

```diff
@@ -81,7 +85,7 @@ public function extract()
     public function reduce(callable $function, $accumulator)
     {
         foreach ($this->value as $item) {
-            $accumulator = call_user_func($function, $accumulator, $item);
+            $accumulator = $function($accumulator, $item);
```

Comment:
Will this work when `$function` will be callback in form of array ie.`[$some, 'action]`?
I don't recall in which PHP version this required `call_user_func`

Reply:
```php
require 'vendor/autoload.php';
$a = ['Widmogrod\Primitive\Listt', 'of'];
$l = $a(12);
var_dump($l);
/*
class Widmogrod\Primitive\Listt#2 (1) {
  protected $value =>
  array(1) {
    [0] =>
    int(12)
  }
}
*/
```

Reply:
Thank you. 

Then this is probably something that works from PHP7.
I have created a ticket to apply it everywhere. 

https://github.com/widmogrod/php-functional/issues/70